experiments/compositions/composition/api/v1alpha1/expanderversion_types.go

```diff
@@ -36,6 +36,12 @@ const (
 	ExpanderTypeGRPC ExpanderType = "grpc"
 )
 
+type ExpanderConfigGVK struct {
+	Group   string `json:"group"`
+	Version string `json:"version"`
+	Kind    string `json:"kind"`
```

Comment:
The Kind here is similar to expander type? like Getter/Jinja?

Reply:
No this is GVK GVR kubernetes resource, kind